CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+### How to make a clean Pull Request:
+
+1. Make sure your pull request has a correct title. For example in PlatformUI we use 'EZP-XXXXX: Title of the pull request'
+2. In the description of the pull request:
+⋅⋅* First add the JIRA link of your story.
+⋅⋅* Then add a short description about what's in the pull request.
+⋅⋅* Sometimes a screenshot, or a screencast is a good way to show your work.
+⋅⋅* Don't forget to mention if your code is tested (manual and unit tests, coverage...).
+3. Make a last check of the code you've just pushed, for example you can check:
```

Comment:
sure, but that's why it's a 'last' check... Anyway i'll add 'again at the end of the sentence.